docs/quickstart.rst

```diff
@@ -27,22 +33,34 @@ Change ``/hello`` and ``/world`` to some actual paths on the website/service you
     [2021-07-24 09:58:46,215] .../INFO/locust.main: Starting web interface at http://0.0.0.0:8089
     [2021-07-24 09:58:46,285] .../INFO/locust.main: Starting Locust |version|
 
+Congratulations, you've run a Locust test!
+
+.. TODO you can't run a test first, and open the web interface to connect to that right? You either need to run it from the web interface, or specify the details and run in headless
+
+Open the :ref:`web interface <interface>` to view the test results, or restart locust with the ``--headless`` parameter to view them on the :ref:`command line <headless>`.
+
+.. _interface:
+
 Locust's web interface
 ======================
 
-Open http://localhost:8089
+Locust's web interface is available by default on http://localhost:8089 . Specify some basic test parameters and a host to test, and once it's started, get reports on the running tests.
```

Comment:
"Specify some basic test parameters and a host to test" is a bit vague? This is a quickstart guide so I'd like it to be a little more specific :)